packages/design/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jobber/design",
-  "version": "0.9.6",
+  "version": "0.9.7",
```

Comment:
Not sure if I had to actually do this. Just double checking. Either way we will need to get this into Jobber fairly quickly...